src/greynirseq/nicenlp/criterions/pos_criterion.py

```diff
@@ -9,34 +9,6 @@
 import torch
 import torch.nn.functional as F
 
-try:
-    from icecream import ic
-
-    ic.configureOutput(includeContext=True)
-except ImportError:  # Graceful fallback if IceCream isn't installed.
-    ic = lambda *a: None if not a else (a[0] if len(a) == 1 else a)  # noqa
-
-from greynirseq.ner.ner_f1_stats import EvalNER
-import greynirseq.nicenlp.utils.greynir.greynir_utils as greynir_utils
-
-import pyximport
-
-pyximport.install()
-import greynirseq.nicenlp.utils.greynir.tree_dist as tree_dist
-
-
-def targets_to_flat_mask_no_bos(targets, pad_idx):
-    bsz, _ = targets.shape
-    padding_mask = torch.cat(
-        [targets.new_empty(bsz, 1).fill_(pad_idx + 1), targets], 1
-    ).eq(pad_idx)
-
-    no_bos_mask = torch.cat([targets.new_empty(bsz, 1).fill_(pad_idx), targets], 1).ne(
-        pad_idx
-    )
-
-    return no_bos_mask[padding_mask.bitwise_not()]
-
 
 @register_criterion("pos_ice")
 class POSCriterion(FairseqCriterion):
```

Comment:
Nothin pos specific in this criterion?

Reply:
Agreed, changing name to MultiLabelTokenClassificationCriterion, along with its filename.